src/index.html

```diff
@@ -11,6 +11,25 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action=""
+      method="post"
+      data-qa="keypress">
```

Comment:
Remember about proper code style, fix it everywhere
```suggestion
      data-qa="keypress"
    >
```